carsFilter/src/main/java/tkhal.filter.controller/MessageFilterController.java

```diff
@@ -0,0 +1,50 @@
+package tkhal.filter.controller;
+
+import model.Car;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.Producer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+
+public class MessageFilterController {
+
+    private final static Logger LOGGER = LoggerFactory.getLogger(MessageFilterController.class);
+    private static Car tempCar = null;
+    private static KafkaConsumer<String, Car> consumer;
+    private static Producer<String, Car> producer;
+    private static FilterKafkaConsumer filterKafkaConsumer = new FilterKafkaConsumer();
+    private static FilterKafkaProducer filterKafkaProducer = new FilterKafkaProducer();
+    private static CarsFilter carsFilter = new CarsFilter();
```

Comment:
very bad practice to instantiate classes like this.
Use constructor for it

Reply:
done